include/z64transition.h

```diff
@@ -23,6 +23,11 @@ typedef struct {
     /* 0xDC */ u16* zBuffer;
 } TransitionUnk; // size = 0xE0
 
+typedef enum {
+    /* 1 */ TRANS_TYPE_FADE_IN = 1,
+    /* 2 */ TRANS_TYPE_FADE_OUT
+} TransitionTypeFade;
```

Comment:
This is too similar to the existing `TransitionType` enum

The current transition docs refer to the `transitionCtx`-based transitions (`TransitionCircle_`, `TransitionTriforce_`, ...) as "instance transitions", which maybe could be reused here

```suggestion
    /* 1 */ TRANS_INSTANCE_TYPE_FADE_IN = 1,
    /* 2 */ TRANS_INSTANCE_TYPE_FADE_OUT
} TransitionInstanceTypeFade;
```
:shrug:

Reply:
This seems fine to me.